src/NuGet.Services.V3/Registration/Models/RegistrationContainerContext.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Protocol.Catalog;
+
+namespace NuGet.Protocol.Registration
+{
+    public class RegistrationContainerContext
+    {
+        [JsonProperty("@vocab")]
+        public string Vocab { get; set; }
+
+        [JsonProperty("catalog")]
+        public string Catalog { get; set; }
+
+        [JsonProperty("xsd")]
+        public string Xsd { get; set; }
+
+        [JsonProperty("items")]
+        public ContextTypeDescription Items { get; set; }
+
+        [JsonProperty("commitTimeStamp")]
+        public ContextTypeDescription CommitTimestamp { get; set; }
```

Comment:
Are all these properties supposed to be of type ContextTypeDescription?

Reply:
Yes, this is all the `@context` goo at the bottom of our JSON files:
![image](https://user-images.githubusercontent.com/94054/68332386-90eb1100-008b-11ea-96d4-880a3b8357b6.png)